gflownet/envs/crystals/composition.py

```diff
@@ -333,12 +344,18 @@ def step(self, action: Tuple[int, int]) -> Tuple[List[int], Tuple[int, int], boo
             if self.get_mask_invalid_actions_forward()[-1]:
                 valid = False
             else:
-                if self._can_produce_neutral_charge():
-                    self.done = True
-                    valid = True
-                    self.n_actions += 1
-                else:
-                    valid = False
+                # TODO: re-enable charge check
+                # Currently enabling it causes errors when training combined
+                # Crystal env, and very significantly increases training time.
+                # if self._can_produce_neutral_charge():
+                #     self.done = True
+                #     valid = True
+                #     self.n_actions += 1
+                # else:
+                #     valid = False
```

Comment:
How replacing the commented out piece of code by
```
if self.do_charge_check:
  # do charge check
```
and `do_charge_check` is control by config?